src/libs/metadata_view.c

```diff
@@ -337,8 +337,14 @@ static void _metadata_view_update_values(dt_lib_module_t *self)
       sqlite3_stmt *stmt;
       DT_DEBUG_SQLITE3_PREPARE_V2(dt_database_get(darktable.db), "SELECT imgid FROM main.selected_images LIMIT 1",
                                   -1, &stmt, NULL);
-      if(sqlite3_step(stmt) == SQLITE_ROW) mouse_over_id = sqlite3_column_int(stmt, 0);
+      if(sqlite3_step(stmt) == SQLITE_ROW) { mouse_over_id = sqlite3_column_int(stmt, 0); }
```

Comment:
this doesn't follow the style in other dt parts...

Reply:
Maybe...but this is more secured if someone modify it later.
As you want, I will revert it.